docs/pages/faq.mdx

```diff
@@ -123,6 +123,18 @@ Teleport provides security-critical support for the current and two previous rel
 See our [Upgrading](./setup/operations/upgrading.mdx) guide for more
 information.
 
+## Which operating systems are supported?
+
+Teleport runs on Linux, macOS, and Windows (client only).
+
+|Operating System| Version       |
+|----------------|---------------|
+| macOS          | X             |
```

Comment:
@codingllama What's the minimum version of macOS `tsh` now supports?

Reply:
macOS 10.13, aka High Sierra.

Source: https://github.com/gravitational/teleport/blob/80baab8ce584bb903ccec566d037a816770d6f81/lib/auth/touchid/api_darwin.go#L20